microsoft/testsuites/xdp/functional.py

```diff
@@ -429,7 +429,7 @@ def verify_xdp_remove_add_vf(self, node: Node, log: Logger) -> None:
         """,
         priority=3,
     )
-    def verify_xdp_community_test(self, node: Node) -> None:
+    def verify_xdp_subtest(self, node: Node) -> None:
```

Comment:
This doesn't need to be changed. Because it's really "community" tests, not our customized tests. Only when we talk the test cases inside of LISA, there are test results and subtest results.

Reply:
Missed changing it back. fixed it!